pkg/reconciler/v1alpha1/taskrun/resources/pod.go

```diff
@@ -43,7 +43,7 @@ import (
 	"github.com/tektoncd/pipeline/pkg/reconciler/v1alpha1/taskrun/entrypoint"
 )
 
-const workspaceDir = "/workspace"
+const WorkspaceDir = "/workspace"
```

Comment:
If these are exported, they should also have descriptive comments, since they'll show up in godoc, and in IDE integrations.

Reply:
and please include in the comment some more info about why each value needs to be used externally :pray: